flink-clients/src/main/java/org/apache/flink/client/program/StreamContextEnvironment.java

```diff
@@ -55,24 +69,67 @@
     private final boolean suppressSysout;
 
     private final boolean enforceSingleJobExecution;
+    private final CheckpointConfig originalCheckpointConfig;
+    private final ExecutionConfig originalExecutionConfig;
+    private final Configuration originalConfiguration;
 
     private int jobCounter;
 
+    private final Collection<JobValidationError> errors;
```

Comment:
Afaik this field is unneeded.

Reply:
It is set in the ctor with the forbidden configuration coming from the instantiation of the StreamEnvironment i.e. `StreamExecutionEnvironment.getExecutionEnvironment(Configuration config)`